charts/stable/homarr/templates/_secrets.tpl

```diff
@@ -0,0 +1,9 @@
+
+{{- $secretName := (printf "%s-homarr-secrets" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
+{{- $secret := randAlphaNum 32 -}}
+{{- with (lookup "v1" "Secret" .Release.Namespace $secretName) -}}
+  {{- $secret = index .data "NEXTAUTH_SECRET" | b64dec -}}
+{{- end }}
+enabled: true
+data:
+  NEXTAUTH_SECRET: {{ $secret }}
```

Comment:
```suggestion
  NEXTAUTH_SECRET: {{ $secret }}
{{- end -}}
```

Reply:
Ohhh I see what I did I copied your small change over the whole file lol